### PR TITLE
Added --in-place flag to deploy script.

### PR DESCRIPTION
**Pull request by computer-whisperer**
This gives the option for deploying changes without deleting or re-starting the robot code. This is primarily for robot programs capable of reloading themselves.

If you don't think it belongs here, that's fine.

### CODE REVIEW

File: lib/pyfrc/mains/cli_deploy.py [hunk 1]
```diff
@@ -80,21 +83,29 @@ def run(self, options, robot_class, **static_options):
...
+            'del_cmd': del_cmd,
             'deploy_dir': deploy_dir,
-            'py_deploy_dir': py_deploy_dir,
             'cmd': deployed_cmd,
             'cmd_fname': deployed_cmd_fname,
             'extra_cmd': extra_cmd
         }
-        
+
+        print(sshcmd)
```
Comment (virtuald) [inline lib/pyfrc/mains/cli_deploy.py]: Intentional?

**Comment by virtuald** [comment]
This looks fine to me.

File: lib/pyfrc/mains/cli_deploy.py (hunk 1 above)
Comment (computer-whisperer) [inline lib/pyfrc/mains/cli_deploy.py]: Oops! I was using that for debugging, and forgot to remove it. :L

**Comment by virtuald** [comment]
I rebased this and merged it in e63931d.

**Pull request closed without merging**